odoo/tools/translate.py

```diff
@@ -1090,6 +1091,14 @@ def _export_imdinfo(self, model: str, imd_per_id: dict[int, ImdInfo]):
                     _logger.exception("Failed to extract terms from %s %s", xml_name, name)
                     continue
                 for term_en, term_langs in translation_dictionary.items():
+                    term_en_unescaped = html_stdlib.unescape(term_en)
+                    value_en_unescaped = html_stdlib.unescape(value_en)
```

Comment:
I understand those could lead to a security issue but I just use it to do comparison. I don't add it in the DB nor a XML rendered to a client so it should be OK.